plugins/discourse-narrative-bot/db/migrate/20230227050146_update_narrative_badge_icons.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class UpdateNarrativeBadgeIcons < ActiveRecord::Migration[7.0]
+  def change
+    execute <<~SQL
+      UPDATE badges
+      SET icon = 'stamp'
+      WHERE
+        name IN ('#{DiscourseNarrativeBot::NewUserNarrative::BADGE_NAME}', '#{DiscourseNarrativeBot::AdvancedUserNarrative::BADGE_NAME}')
```

Comment:
I have one more change request :wink: 

```suggestion
      WHERE
        name IN ('#{DiscourseNarrativeBot::NewUserNarrative::BADGE_NAME}', '#{DiscourseNarrativeBot::AdvancedUserNarrative::BADGE_NAME}')
        AND icon = 'fa-certificate'
```